src/helpers/get_metrics_from_df.py

```diff
@@ -33,20 +41,21 @@ def get_metrics_from_df(df, target_column: str, required_metrics: list = ["mae",
     mse_values = [] if require_mse else None
     mape_values = [] if require_mape else None
 
-    # Calculate metrics for each forecast column compared to the target column
-    for col in df.columns[1:]:
-        if require_mae:
-            mae = mean_absolute_error(df[target_column], df[col])
-            mae_values.append(mae)
-        if require_mse:
-            mse = mean_squared_error(df[target_column], df[col])
-            mse_values.append(mse)
-        if require_mape:
-            mape = mean_absolute_percentage_error(df[target_column], df[col])
-            mape_values.append(mape)
+    # Calculate metrics for each forecast column compared to the observation column
+    for col in df.columns:
+        if col != observation_column:
```

Comment:
Can we just do a `df[observation_column]` to remove this `for` loop?

Reply:
Sorry, I misread. But I think it can still be simplified using the .drop() function

Reply:
Actually forget this, not gonna copy the df to do the dropping

Reply:
Resolved.